rest-server/src/controllers/auth.js

```diff
@@ -24,16 +24,16 @@ const logger = require('../config/logger');
 
 
 /**
- * Login.
+ * Get the token.
  */
-const login = (req, res) => {
+const getToken = (req, res) => {
   const username = req.body.username;
   const password = req.body.password;
   const expiration = req.body.expiration;
   authModel.check(username, password, (err, state, admin) => {
     if (err || !state) {
       logger.warn('user %s authentication failed', username);
-      return res.status(401).json({
+      return res.status(500).json({
```

Comment:
UNAUTHORIZED instead of INTERNAL SERVER ERROR

Reply:
ok. Will change.